benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/SparkUtil.scala

```diff
@@ -98,6 +107,29 @@ trait SparkUtil {
     sparkSession.close()
   }
 
+  private def copyResourceToDir(
```

Comment:
I would suggest to put this function into `ResourceUtil`.

Reply:
Done.